hoodie-client/src/main/java/com/uber/hoodie/WriteStatus.java

```diff
@@ -45,8 +45,8 @@
 
   private HoodieWriteStat stat = null;
 
-  private long totalRecords = 0;
-  private long totalErrorRecords = 0;
+  protected long totalRecords = 0;
```

Comment:
Can you add setters for these 2 attributes instead of changing access restrictions ?

Reply:
so add both setters and getters? LGTM.